spi-api/src/main/java/de/adorsys/aspsp/xs2a/spi/domain/account/SpiAccountDetails.java

```diff
@@ -33,7 +33,7 @@
     @Id
     @Setter
     @NonFinal
-    private String id;
+    private String id; // TODO shouldn't this be resourceId?
```

Comment:
The name of the field was taken from the 1.0 version of the specification. We will rename it in the next version of our project according to specification 1.1